formspecs.lua

```diff
@@ -34,7 +36,9 @@ function travelnet.show_current_formspec(pos, meta, player_name)
 	local formspec = meta:get_string("formspec") ..
 		("field[20,20;0.1,0.1;pos2str;Pos;%s]"):format(minetest.pos_to_string(pos))
 	-- show the formspec manually
-	minetest.show_formspec(player_name, travelnet_form_name, formspec)
+	minetest.after(0.1, function ()
```

Comment:
```suggestion
	minetest.after(0, function ()
```
To make it more clear that action is meant to be executed immediately on next globalstep and adding delay is not really intended.